lesson3/home_task_queries.sql

```diff
@@ -0,0 +1,171 @@
+/*
+ Завдання на SQL до лекції 02.
+ */
+
+
+/*
+1.
+Вивести кількість фільмів в кожній категорії.
+Результат відсортувати за спаданням.
+*/
+-- SQL code goes here...
+select 
+	c."name",
+	count(*)
+from public.film_category fc
+left join category c 
+on fc.category_id = c.category_id 
+group by 1
```

Comment:
please, never use 1, 2 etc, use explicit names, if you don't want to get an unexpected surprise